fastlane/Fastfile

```diff
@@ -980,8 +980,10 @@ platform :ios do
       ],
       exclude: [
         '*Vendor*',
+        '**/AppLocalizedString.swift',
```

Comment:
We don't want to localize the localization routine as that would create issues.

This makes me wonder if we should update the format for the routines to:

```ruby
routines: [
  { name: 'AppLocalizedString', path: 'path/to/file.swift' }
],
```

And then handle excluding that path within the toolkit. 

_Gold plating..._

Reply:
I think that's a laudable idea but might not be flexible enough to all our codebases that are not as clean as WP or WC and might e.g. still want to adopt the linter and also have a custom routine already in place, but just not in a dedicated `.swift` file containing only that. etc.

So in the end it might end up coupling the routine name and its assumption of it being isolated into a single file a bit too much. Besides, in some codebases or in the future, the file where the routine is declared might end up being already excluded because provided by a SPM plugin or a Pod that would already be excluded from parsing 🤷 

Ultimately I'm not sure it's worth doing a breaking change (due to parameter type change) on the release-toolkit just for this 😅